crates/bcr-ebill-transport/src/notification_service.rs

```diff
@@ -1219,7 +1209,8 @@ mod tests {
 
     fn check_chain_payload(event: &EventEnvelope, bill_event_type: BillEventType) -> bool {
         let valid_event_type = event.event_type == EventType::Bill;
-        let event: Result<Event<BillChainEventPayload>> = event.clone().try_into();
+        let event: bcr_ebill_core::protocol::Result<Event<BillChainEventPayload>> =
```

Comment:
why is this type fully qualified and not the others? Is there a naming conflict?